post-processor/googlecompute-import/post-processor.go

```diff
@@ -0,0 +1,204 @@
+package googlecomputeimport
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/storage/v1"
+
+	"github.com/hashicorp/packer/builder/googlecompute"
+	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/helper/config"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/post-processor/compress"
+	"github.com/hashicorp/packer/template/interpolate"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/jwt"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+
+	Bucket            string            `mapstructure:"bucket"`
+	ImageDescription  string            `mapstructure:"image_description"`
+	ImageFamily       string            `mapstructure:"image_family"`
+	ImageLabels       map[string]string `mapstructure:"image_labels"`
+	ImageName         string            `mapstructure:"image_name"`
+	ProjectId         string            `mapstructure:"project_id"`
+	AccountFile       string            `mapstructure:"account_file"`
+	KeepOriginalImage bool              `mapstructure:"keep_input_artifact"`
```

Comment:
This should not be here, see https://github.com/hashicorp/packer/blob/master/post-processor/vagrant/post-processor.go#L153 how this should work.

Reply:
I believe you are asking me to get rid of the `keep_input_artifact` configuration option and instead use  `provider.KeepInputArtifact()`. I'm not really super good and Go Lang, so ...

Do I need to implement the entire `PostProcessProvider` function?

https://github.com/hashicorp/packer/blob/master/post-processor/vagrant/post-processor.go#L75

Reply:
Sorry, I'm wrong, you can just ignore this comment.